pages/glossary/clearly-labeled-location.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Clearly labeled location
+key: clearly-labeled-location
+unambiguous: true
+objective: false
+---
+
+Secondary information and alternative controls or functionality are often not displayed together with primary information or functionality. For example, an option to change a web page to dark mode may be placed on an options page instead of being available on every page and page state of a website. Another example is a maps application, where, instead of using GPS, an option is available in a dropdown menu to set the current location of the device. Such content should be placed in a clearly labeled location.
+
+The location of a target is said to be **"clearly labeled"** when the target can be found by activating [user interface components][] whose purpose, indicated by their [label], leads the user to find the target or to another [page][web page] or page state from which this action can be repeated until the target is found.
+
+Whether or not the content is "clearly labeled" depends on the starting point of the search. If page A has a clearly labeled link to some piece of important content, it is clearly labeled. Page B, which can be in the same website, may not have such a link or may have a link with an ambiguous link text, and so the location of the content starting from page B is not clearly labeled.
```

Comment:
"If page A has a clearly labeled link to some piece of important content, it is clearly labeled."
So, if it's clearly labeled, then it's clearly labeled 🤔 

This at least need to be split as in "the link is a clear label" => "the page is clearly labeled".

Reply:
Rephrased.